kolibri/plugins/device/assets/src/views/PageWrappers/AppBarDevicePage.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+
+  <AppBarPage :title="$tr('deviceManagementTitle')">
```

Comment:
More of a question than a request to change - aside from the title string here, is there anything that would actually need to be consolidated in a component here, or could each top level page in device have its own (slightly boilerplatey) top level component that just directly included these things?

Reply:
Just to clarify - are you asking if there is any reason for having this component w/ the string here as opposed to having this template applied to every `*Page` component in the plugin directly? The only other thing this does really is include the proper side nav though. 

re: moving the string to a js translator, this and the DeviceManagementSideNavEntry are the only places we use the message "Device" at the moment and we typically won't make a string common unless it's used 3+ times in the same context

Maybe I'm misunderstanding your question though and would be happy to clarify on a call or in Slack

Reply:
Right, I was just wondering if there was any benefit from having this component instead of just moving the functionality into the top level page components and not having to have these conditionality here.

Reply:
Could do -  then each of them would import DeviceTopNav themselves and they'd just use `AppBarPage` directly and put this somewhere when needed.

```
<template #subNav>
      <DeviceTopNav v-if="!hideSubNav" />
 </template>
```

I can't see any downside to that approach and wish I'd taken it in the first place.

Separately, now looking back I'm wondering why I didn't just have every page put their business in their own KPageContainer rather than adding a prop & conditional here in the first place.

Reply:
Yeah, that was kind of my thought - this felt like it was leaving some residual, Device specific, CoreBase lying around.

Reply:
In practice the difference seems to be that this way does little more than DRY up the importing and implementation of the SubNav bit across a bunch of components -- which could get particularly hairy when refactoring Coach's umpteen pages. 

Pretty sure I can do this refactor pretty quickly with a little bit of vim macro magic.